app.js

```diff
@@ -0,0 +1,103 @@
+var express = require('express');
+var path = require('path');
+var favicon = require('serve-favicon');
+var logger = require('morgan');
+var cookieParser = require('cookie-parser');
+var bodyParser = require('body-parser');
+
+var index = require('./routes/index');
+var users = require('./routes/users');
+
+var app = express();
+
+// view engine setup
+app.set('views', path.join(__dirname, 'views'));
```

Comment:
This gets included w/ express generator, but we don't need any sort of view engine since we're just sending info to the app, so you can remove this line and the one below it.